pkg/addons/addons_test.go

```diff
@@ -28,12 +28,21 @@ import (
 	"k8s.io/minikube/pkg/minikube/localpath"
 )
 
-func createTestProfile(t *testing.T) string {
+func createTestProfile(t *testing.T, status bool) string {
```

Comment:
Adding the `defer os.RemoveAll(td)` to this function means that the temporary directory that we create at the start of the function will be removed as soon as `createTestProfile` finishes executing -- so right now   this function both creates and deletes the temp home directory

We probably want this function to return the home directory, and `defer os.RemoveAll(td)` in the tests that call this function instead

```go
func createTestProfile(t *testing.T) (string, string)
```